### PR TITLE
The remaining time is now independent of the browser's system clock.

### PR DESCRIPTION
**Pull request by afzaledx**

### CODE REVIEW

**Comment by afzaledx** [comment]
@muhhshoaib

**Comment by muhhshoaib** [comment]
I have tried it on setting the time on my local server and gave exam from different machine using my IP address and reset the time to 1 hour earlier and it is working fine.

Looked at the code :+1: for me.

File: edx_proctoring/static/proctoring/spec/proctored_exam_spec.js [hunk 1]
```diff
@@ -46,31 +46,36 @@ describe('ProctoredExamView', function () {
...
+                JSON.stringify({
+                    time_remaining_seconds: -10
+                })
+            ]
+        );
+        this.proctored_exam_view.timerTick = 4; // to make the ajax call.
+        var reloadPage = spyOn(this.proctored_exam_view, 'reloadPage');
+        this.proctored_exam_view.updateRemainingTime(this.proctored_exam_view);
+        this.server.respond();
+        this.proctored_exam_view.updateRemainingTime(this.proctored_exam_view);
```
Comment (chrisndodge) [inline edx_proctoring/static/proctoring/spec/proctored_exam_spec.js]: Thanks for the Jasmine unit tests

**Comment by chrisndodge** [comment]
+1. Feel free to merge this in...

**Pull request merged**